tree/ntuple/v7/inc/ROOT/RColumn.hxx

```diff
@@ -195,25 +195,27 @@ public:
    /// For offset columns only, look at the two adjacent values that define a collection's coordinates
    void GetCollectionInfo(const NTupleSize_t globalIndex, RClusterIndex *collectionStart, ClusterSize_t *collectionSize)
    {
-      auto idxStart = (globalIndex == 0) ? 0 : *Map<ClusterSize_t, EColumnType::kIndex>(globalIndex - 1);
-      auto idxEnd = *Map<ClusterSize_t, EColumnType::kIndex>(globalIndex);
+      auto idxStart = *Map<ClusterSize_t, EColumnType::kIndex>(globalIndex);
+      auto idxEnd = *Map<ClusterSize_t, EColumnType::kIndex>(globalIndex + 1);
       auto selfOffset = fCurrentPage.GetClusterInfo().GetIndexOffset();
       if (globalIndex == selfOffset) {
          // Passed cluster boundary
-         idxStart = 0;
+         // idxStart = 0;
```

Comment:
I think the issue is here, but I'm not sure what we should set idxStart to?

Reply:
I think we can just drop the `if` clause altogether.